<!DOCTYPE html
	PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
<title>HVDRP Mutation Reporter Tool FAQ</title>
<link rel="stylesheet" type="text/css" href="/res/hvdr.css"/>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"/>
<style type="text/css">
p.q { background: lightblue; font-style: italic; font-weight: bold; padding:0.5em; }
p.a { margin-bottom:2.5em; font-style: normal; font-weight: normal; }
</style>
</head>
<body style="background:#ffff99;">

<h1>HVDRP Mutation Reporter Tool</h1>
<h2>Frequently Asked Questions</h2>

<div style="margin:3em;">

<p class="q">Is this tool free of charge?</p>
<p class="a">Yes, this tool may be used by anyone, free of charge.</p>

<p class="q">The site is slow or inaccessible. Is a mirror available?</p>
<p class="a">The tool is hosted by the University of the Witwatersrand and under normal circumstances, should always be available and responsive. Please contact us if you continue to have difficulties in this regard. As this tool is under active development, no mirrors are available.</p>

<p class="q">Can I download the tool to use offline?</p>
<p class="a">As the tool is one component of a larger, integrated project, and follows the client-server model, it is not available for download as a standalone application. If you have a specific need in this regard, please contact us.</p>

<p class="q">What data has been used to test and develop the tool?</p>
<p class="a">The tool has been tested extensively with nucleotide sequence data from the hepatitis B virus, but data from any source can be processed.</p>

<p class="q">Is a paper available? How should I cite the tool?</p>
<p class="a">If you have used data or output generated by the tool in your publications or presentations, please cite the following paper: <br><br>
<tt><b>Bell, T. G. and Kramvis, A.</b> (2013) <i>Mutation Reporter Tool: an online tool to interrogate loci of interest, with its utility demonstrated using hepatitis B virus</i>. Virology Journal <b>10:</b> 62. <br><br><img src="/res/open-access.gif" style="vertical-align:middle"> Available <a href="http://www.virologyj.com/content/10/1/62" target="_blank">online</a>.</tt>
</p>

<p class="q">What input file formats are supported?</p>
<p class="a">Input files must be plain text in FASTA format, and may contain either aligned nucleotide or aligned amino acid sequence data, from one or more samples.</p>

<p class="q">I have sequence data in program X. How can I convert it to FASTA format?</p>
<p class="a">Most bioinformatics programs should be able to export sequence data in FASTA format. In the exceptional situation that this is not the case, it may be possible to export the data into another program, which can export FASTA files.</p>

<p class="q">Why must input files be in FASTA format? Why can't I upload files in format X?</p>
<p class="a">FASTA is a universal bioinformatics file format. To ensure maximum compatability, we have selected this standard format for input files. More information on the FASTA file format is available at <a href="http://en.wikipedia.org/wiki/FASTA_format" target="_new">Wikipedia</a>.</p>

<p class="q">What are the anchor motif and anchor position? Why are they required?</p>
<p class="a">The tool searches for the first occurrence of the anchor motif in the first sequence in the input file. The position at which the anchor motif starts is given the co-ordinate of the anchor position. This co-ordinate mapping allows the tool to find the positions of the specified loci within the sequence fragment data. The loci specified would typically be numbered according to their co-ordinates in the genome, whereas the sequence data would typically be a subgenomic fragment.</p>

<p class="q">I don't understand the output grouping.</p>
<p class="a">If no output grouping is specified, all the loci are displayed in a single column. The loci of interest can be grouped into logical units by specifying output grouping columns. For example, consider the following ten loci of interest: 1762, 1764, 1809-1812, 1858, 1862, 1888, 1896. The first two loci are interpreted as a unit, as are the next four loci. An output grouping of "2,4,1,1,1,1", for example, would place the data from the first 2 loci into the first column, followed by the data from the next 4 loci into the second column, etc. This allows the data to be analyzed and interpreted more easily.</p>

<p class="q">What is the sequence ID pattern regular expression used for?</p>
<p class="a">Please see the Regular Expression <a href="regexp.html" target="_blank">tutorial</a>.</p>

<p class="q">I don't understand the parameters for the mutation distribution graph.</p>
<p class="a">Please see the Graph <a href="graphtut.html" target="_blank">tutorial</a>.</p>

<p class="q">What is Discovery Mode?</p>
<p class="a">When the tool is used in "Discovery Mode", it examines all the loci specified, but only selects for inclusion those loci at which variation occurs. This can be used to discover positions of interest by excluding positions at which no variation occurs. A single variant residue is sufficient for a locus to be included. When the "Discovery Mode" check-box is checked, input fields for several parameters, which are incompatible with this mode, will be removed from the input page. These fields will reappear when the check-box is unchecked.</p>

<p class="q">What is the motif distribution table and graph on the output page?</p>
<p class="a">For this section of output, all of the specified loci are concatenated into a single motif. The distribution of this motif in the input data is tabulated and displayed as a graph.</p>

</div>

</body>

</html>

